src/main/java/org/apache/commons/lang3/ClassUtils.java

```diff
@@ -982,6 +982,59 @@ public static String getShortCanonicalName(final String canonicalName) {
         return getShortClassName(getCanonicalName(canonicalName));
     }
 
+    /**
+     * Check if the given {@code Method} is overridable in the given target {@code Class}.
+     *
+     * <p>This method returns {@code false} for any {@code null} input or method with the static or final modifiers.</p>
+     * <pre>
+     * <code>
+     *   private static class ABG {
+     *        private void privateMethod() {
+     *        }
+     *        void defaultMethod() {
+     *        }
+     *        public void publicMethod() {
+     *        }
+     *        public final void finalMethod() {
+     *        }
+     *        public static void  staticMethod() {
+     *        }
+     *    }
+     *   Method publicMethod  = ABG.class.getDeclaredMethod("publicMethod", null)
+     *   Method privateMethod = ABG.class.getDeclaredMethod("privateMethod", null)
+     *   Method defaultMethod = ABG.class.getDeclaredMethod("defaultMethod", null)
+     * </code>
+     *
+     * ClassUtils.isOverridable(null, null)                = false
+     * ClassUtils.isOverridable(null, Class)               = false
+     * ClassUtils.isOverridable(Class, null)               = false
+     * ClassUtils.isOverridable(privateMethod, ABG.class)) = false
+     * ClassUtils.isOverridable(defaultMethod, ABG.class)) = true
```

Comment:
Update this to `packagePrivateMethod`

The limitations of the result for packagePrivateMethod should be documented. For example:

```
<p>Note that the result of {@code isOverridable} when called on a package-private method is
sensitive to the package of the target class. When invoked on the declaring class or a child
class within the same package the result will be {@code true} as the method is visible within 
the package. When invoked on a child class within a different package the result will be
{@code false}.
```